src/radical/entk/execman/rp/task_processor.py

```diff
@@ -443,12 +456,18 @@ def create_cud_from_task(task, placeholders, prof=None):
         cud.sandbox    = task.sandbox
         cud.post_exec  = task.post_exec
 
+        # FIXME: Use tags properly.
```

Comment:
I don't understand why the tag resolver is not used - is it not complete, yet?

Reply:
Oops, I just remembered that task names are not unique.

Reply:
Fixed